lib/statsd_middleware.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+class StatsdMiddleware
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    status, headers, response = @app.call(env)
+    params = env['action_dispatch.request.path_parameters']
+    key = "api.external.request#status=#{status},controller=#{params[:controller]},action=#{params[:action]}"
```

Comment:
Interesting. I've also never seen key=value command delimited StatsD metrics either. If this is going to a key-value system and just using the StatsD name or transport, 👍 'cause order doesn't matter.